src/Sly/NotificationPusher/Adapter/Apns.php

```diff
@@ -134,6 +134,7 @@ public function getServiceMessageFromOrigin(DeviceInterface $device, BaseOptione
         ;
 
         $sound = $message->getOption('sound', 'bingbong.aiff');
+        $contentAvailable = $message->getOption('content-avaibile');
```

Comment:
Hi, thanks for your PR.
Here is a typo with using `content-avaibile` instead of `content-available`.